src/main/java/org/generationcp/ibpworkbench/ui/WorkbenchMainView.java

```diff
@@ -551,7 +552,6 @@ void addAdminButton(final HorizontalLayout layout) {
 
 	}
 
-	@PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_SUPERADMIN')")
 	void layoutAddProgramButton(final HorizontalLayout layout) {
 
 		if (Boolean.parseBoolean(this.isAddProgramEnabled)) {
```

Comment:
Discussed with @mcrimi that we can delete "desktop" toggling properties since we deprecated desktop version. The
**workbench.is.add.program.enabled** doesn't make sense now that we have permissions configuration for that.

Logged technical backlog ticket: https://ibplatform.atlassian.net/browse/IBP-3009